docs/core/examples/mouse.py

```diff
@@ -60,7 +60,7 @@ def up_right(self):
         log.msg("right")
 
     def move(self, x, y):
-        log.msg("(%d,%d)" % (x, y))
+        log.msg("({},{})".format(x, y))
```

Comment:
`t.p.mice` won't be ported to Python 3, so these changes aren't necessary.  In fact, since it's deprecated, maybe we should remove this example?